Docs -- designing  design kit v2 (#1686)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/designing/design-kit.mdx

Inside of a design file, navigate to the **Main menu panel** in the top left of the toolbar (A). Open the menu and select **Libraries** from the list (B).

<Row>
<Column  colLg={8} colMd={8} colSm={4}>

![Adding components from a library](../../images/designing/design-kit/design-kit-I.png)

</Column>
</Row>

Then in the Libraries modal that appears, you can search for the library under Carbon for IBM.com (C) and click the **Add to file** button next to the `Carbon for IBM.com v2 BETA` (D).

<Row>
<Column  colLg={8} colMd={8} colSm={4}>

![Adding components from a library](../../images/designing/design-kit/design-kit-II.png)

</Column>
</Row>

To preview the v2 library, visit the following view-only link.

<Row className="resource-card-group">
<Column colLg={4} colMd={4} noGutterSm>
  <ResourceCard
    subTitle="Carbon for IBM.com v2 Figma library"
    href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-design-kit">

![Figma icon](../../images/icon/figma.svg)

</ResourceCard>
</Column>
</Row>

### Figma tutorials

> If you are new to Figma, they offer <a href="https://www.youtube.com/c/Figmadesign/playlists?view=50&sort=dd&shelf_id=3" target="_blank" rel="noopener noreferrer">video tutorials</a> on their official YouTube channel.

Reviewer: ```suggestion
If you are new to Figma, tutorials are offered on their official <a href="https://www.youtube.com/c/Figmadesign/playlists?view=50&sort=dd&shelf_id=3" target="_blank" rel="noopener noreferrer">YouTube channel</a>.
```

### Figma guild

> _For IBMers only_: Sign up for the <a href="https://ec.yourlearning.ibm.com/w3/series/10200299" target="_blank" rel="noopener noreferrer">Figma Guild Series in YourLearning</a> to stay up to date on future Figma Events at IBM. Figma Fridays offer in-depth tutorials and workshops for IBM employees. Recordings are available on the <a href="https://secure.video.ibm.com/channel/23570833/playlist/641336" target="_blank" rel="noopener noreferrer">Figma Fridays playlist</a> on Watson Media.

Reviewer: ```suggestion
_For IBMers only_: Sign up for the <a href="https://ec.yourlearning.ibm.com/w3/series/10200299" target="_blank" rel="noopener noreferrer">Figma Guild Series</a> in YourLearning to stay up to date on future Figma Events at IBM. Figma Fridays offer in-depth tutorials and workshops for IBM employees. Recordings are available on the <a href="https://secure.video.ibm.com/channel/23570833/playlist/641336" target="_blank" rel="noopener noreferrer">Figma Fridays playlist</a> on Watson Media.
```

## Other Figma libraries

> While the Carbon for IBM.com library can be used on its own, there are other libraries in the IBM organization that are useful for designing web pages. You can enable these inside your Figma file, or visit these view-only links.

Reviewer: ```suggestion
While the Carbon for IBM.com library can be used on its own, there are other libraries in the IBM organization that are useful for designing web pages. You can enable these inside your Figma file, or visit these links (read-only).
```

To enable the grid, select a frame at the appropriate breakpoint width, and select the **Styles** button under **Layout grid** in the right Design panel. Select the appropriate breakpoint to apply the grid to the selected frame.

To learn more about the 2x grid, visit the Carbon Design system [2x Grid](https://carbondesignsystem.com/guidelines/2x-grid/overview/) page.

### Color styles

The Dotcom Figma library uses color styles from the `(v11) All themes - Carbon Design System` library. To apply a color style, select an object then in the **Styles** menu you can select a style from the Carbon theme libraries or the IBM Design Language libraries. In addition to applying color styles to objects, you can also apply Color styles to Text layers.

To learn more about applying color styles in Figma, see the [Figma docs](https://help.figma.com/hc/en-us/articles/360040316193).

### Text styles

> The Dotcom Figma library uses text styles from the `(v11) Text Styles - IBM Design Language` library. To apply a text style, select a text layer, then in the properties panel, choose a text style from the `(v11) Text styles - IBM Design Language`. Use color styles to change the color of a text style.

Reviewer: ```suggestion
The Dotcom Figma library uses text styles from the `(v11) Text Styles - IBM Design Language` library. To apply a text style, select a text layer, then in the properties panel, choose a text style from `(v11) Text styles - IBM Design Language`. Use color styles to change the color of a text style.
```
